test/diffusions_test.jl

```diff
@@ -1,18 +1,30 @@
-using MatrixNetworks
+import KahanSummation
+using LinearAlgebra
+using SparseArrays
 
-using Base.Test
+using Test
 @testset "diffusions" begin
-    function _normout(P)
+    function _normout(P::SparseArrays.SparseMatrixCSC{T,Int64}) where T
         n = size(P,1)
-        colsums = sum(P,1)
+        colsums = sum(P,dims=1)
         (pi,pj,pv) = findnz(P)
         P = sparse(pi,pj,pv./colsums[pj],n,n)
     end
 
+    function _normout(P::LinearAlgebra.Adjoint{Float64,SparseArrays.SparseMatrixCSC{T,Int64}}) where T
+        n = size(P,1)
+        colsums = sum(P,dims=1)
+        (pi,pj,pv) = begin
+            II = findall(!iszero, P)
```

Comment:
Isn't this going to be really really slow?

Reply:
Ah yes. This was because findnz is not supported on adjoint (and on full matrices as a matter of fact) -- what I have here is what you'd do for a full matrix :(

Here's how I'm fixing it: `pj,pi,pv = findnz(P.parent)` -- note the pj and pi are flipped

Reply:
Fixed here e78c50bb1ca0413f5207c6f904ee066672d4ee99